app/src/main/res/layout/dashboard_list_item_category.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+        <import type="androidx.databinding.ObservableField"/>
+        <import type="androidx.lifecycle.LiveData"/>
+        <import type="org.cru.godtools.base.ui.util.ModelUtils" />
+        <import type="java.util.Locale" />
+
+        <variable name="category" type="String" />
+        <variable name="selectedCategory" type="LiveData&lt;String>" />
+        <variable name="callbacks" type="ObservableField&lt;org.cru.godtools.ui.dashboard.tools.CategoryAdapterCallbacks>"/>
+        <variable name="primaryLanguage" type="LiveData&lt;Locale&gt;" />
+    </data>
+
+    <com.google.android.material.card.MaterialCardView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_margin="4dp"
+        android:onClick="@{() -> callbacks.onCategorySelected(category)}"
+        android:padding="6dp"
+        app:cardBackgroundColor="@color/white"
+        app:cardCornerRadius="@dimen/card_corner_radius"
+        app:cardElevation="4dp"
+        app:rippleColor="@color/gt_blue"
+        app:strokeColor="@{category == selectedCategory ? @color/gt_blue : @color/white}"
+        app:strokeWidth="@{category == selectedCategory ? 10f : 0f}">
```

Comment:
the border should be `2dp` and not `10px`

Reply:
A little confused I thought I had to use a float.

Reply:
no, you should be able to let the binding expression resolve the dimen constant for you and pass that value to the appropriate binding adapter.

Reply:
this should work, or something very similar to it

```
app:strokeWidth="@{category == selectedCategory ? @dimen/card_border_size : 0}"
```